demos/serverless/src/handlers.js

```diff
@@ -127,7 +127,13 @@ exports.logs = async (event, context) => {
     });
   }
   putLogEventsInput.logEvents = logEvents;
-  await cloudWatchClient.putLogEvents(putLogEventsInput).promise();
+  await cloudWatchClient.putLogEvents(putLogEventsInput).promise().then(() => {
+    console.log('Successfully written to cloudwatch using token ' + uploadSequence + ' and the LogEvent:');
```

Comment:
In the description, you said:
> We need to have console logs on the cloudwatch handler side to trace errors if any

Do we need to log messages for the successful case?